store/store.go

```diff
@@ -101,7 +101,7 @@ func (bs *BlockStore) LoadPartSet(height int64) (*types.PartSet, *types.BlockMet
 		if part == nil {
 			return nil, nil, fmt.Errorf("block part not found")
 		}
-		wasAdded, err := partSet.AddPart(part)
+		wasAdded, err := partSet.AddPartWithoutProof(part)
```

Comment:
this is not verifying the proofs in all cases?

Reply:
which cases need to be verified?

Reply:
we can remove this, as it was just an observation from looking at the profiles. This step is a noticable chunk of cpu and allocations

Reply:
so we assume the parts are either recovered from the mempool or verified in the block prop reactor? so we can trust them at this level?

Reply:
yeah, we should never commit a block that has an invalid part